README.md

```diff
@@ -34,15 +38,51 @@ end
 
 ### Configuration
 
-We need to manually load DNS records to validate if a MX exists or not. When
-we load the library Erlang doesn't have its DNS record list yet. So to avoid
-any problem, we define a default DNS. By default the value is : `4.2.2.1`.
+```elixir
+# config/config.exs -- default
+config :email_checker,
+  default_dns: :system,
+  also_dns: [],
+  validations: [Format,MX,SMTP],
```

Comment:
I try to follow https://github.com/niftyn8/elixir_style_guide/blob/master/README.md as much as much possible for code format. 

Could you try to add a space between each comma in your array?